### DISCUSSION TITLE
Backtick doesn't show in key combination

### DISCUSSION THREAD

**Discussion by Susko3** [ppy/osu#15524] [Q&A] [answered]
This is entirely due to font, works properly if using default font. Not sure why fallback isn't working.

![image](https://user-images.githubusercontent.com/16479013/140747112-b48fad80-43a1-4dc5-b6df-1e5c145f5787.png)

**Comment by Susko3** [comment]
This key was previously stringified as the tilde key (the shifted state‽): `~`

**Comment by smoogipoo** [comment, marked as answer]
Definitely something we'll want fixed, as backtick is a pretty common key for aux functions like quick-retry. https://github.com/ppy/osu/issues/15532
